Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM maven:3.5-jdk-8
-ENV MAXWELL_VERSION=1.14.7 KAFKA_VERSION=0.11.0.1
-
-COPY . /workspace
+ENV MAXWELL_VERSION=1.14.7 KAFKA_VERSION=1.0.0
```

Comment:
Is it worth making `KAFKA_VERSION` an `ARG`, so it may be optionally specified in the newly added ` bin/maxwell-docker` script?

Reply:
so this version just specifies the default kafka version that the maven package gets built with.  The runtime version is specified elsewhere (with the command line flag `--kafka_version`)

Now, it'd be quite nice to have a whole host of common env-variables for the docker image, but I think it's outside the scope of this PR